### PR TITLE
Adds new job query fields

### PR DESCRIPTION
**Pull request by hoddr**
start date, completion date, and fail date as new query fields.

### CODE REVIEW

File: voxel51/query.py [hunk 1]
```diff
@@ -246,4 +246,5 @@ def __init__(self):
         '''Initializes a JobsQuery instance.'''
         super(JobsQuery, self).__init__([
             "id", "name", "state", "archived", "upload_date", "analytic_id",
-            "auto_start", "use_gpu"])
+            "auto_start", "use_gpu", "start_date", "completion_date",
```
Comment (brimoor) [inline voxel51/query.py:249]: I just read documentation for `queued_time ` and `execution_time` in the `platform-docs`. We need to be more careful about keeping API + client libs + docs in-sync.

**Review by brimoor — CHANGES_REQUESTED** [review]

File: voxel51/query.py (hunk 1 above)
Comment (hoddr) [reply voxel51/query.py:249]: That is available on the job-statistics query, not the standard `v1/jobs` query (which this is).

Comment (brimoor) [reply voxel51/query.py:249]: Hmm, okay.

**Review by brimoor — APPROVED** [review]
LGTM

**Pull request merged**